app/controllers/static_controller.rb

```diff
@@ -1,4 +1,5 @@
 class StaticController < ApplicationController
```

Comment:
This looks good. I think we should also have a "Current Games" section because after I joined a game I have to enter the id in the path in order to see the game.